tekton/images/ko/Dockerfile

```diff
@@ -16,4 +16,4 @@ LABEL maintainer "Tekton Authors <tekton-dev@googlegroups.com>"
 
 ENV GO111MODULE on
 RUN go get github.com/google/ko/cmd/ko@master
-
+RUN go install sigs.k8s.io/kustomize/kustomize/v3
```

Comment:
This doesn't work here because the golang image does not include `gcc`

Reply:
Yes I had not realised just how minimal the `ko` image, based on Alpine, was. I've had to `apk add` `git`, `musl-dev` and `gcc` to get `kustomize` installed. I hope that's acceptable.